bin/sdl_run

```diff
@@ -161,6 +130,37 @@ if [[ ${PRECOMPILED} != true ]]; then
   echo "Compiling code..."
   sdl_make_tracklooper ${FLAGS} >> ${LSTOUTPUTDIR}/${JOBTAG}__LSTRun.log
 fi
+
+# If a backend is specified then make sure that corresponding library exists
+# and make a symbolic link to the correct binary
+if [ "${BACKEND}" == "cuda" ]; then
+  if [ ! -f ${TRACKLOOPERDIR}/SDL/libsdl_cuda.so ]; then
+    echo "Error: CUDA backend was not compiled."
+    exit 1
+  fi
+  ln -s -f ${TRACKLOOPERDIR}/bin/sdl_cuda ${TRACKLOOPERDIR}/bin/sdl
+elif [ "${BACKEND}" == "cpu" ]; then
+  if [ ! -f ${TRACKLOOPERDIR}/SDL/libsdl_cpu.so ]; then
+    echo "Error: CPU backend was not compiled."
+    exit 1
+  fi
+  ln -s -f ${TRACKLOOPERDIR}/bin/sdl_cpu ${TRACKLOOPERDIR}/bin/sdl
+elif [ "${BACKEND}" == "rocm" ]; then
+  if [ ! -f ${TRACKLOOPERDIR}/SDL/libsdl_rocm.so ]; then
+    echo "Error: ROCM backend was not compiled."
+    exit 1
+  fi
+  ln -s -f ${TRACKLOOPERDIR}/bin/sdl_rocm ${TRACKLOOPERDIR}/bin/sdl
+elif [ "${BACKEND}" == "default" ]; then
+  if [[ ! -L ${TRACKLOOPERDIR}/bin/sdl || ! -e ${TRACKLOOPERDIR}/bin/sdl ]]; then
```

Comment:
isn't `! -L` redundant here (meaning that `-e` covering all kinds of files)?

Reply:
Oh yeah, I'm not sure why I had it like that. I think we should remove the `-e` one since `-L` also checks if the link is valid.

Reply:
Nevermind, `-L` doesn't check if it's valid. I guess I was using `-L` to check if it's a link and `-e` to check if it's valid. But `-e` should be enough.